### DISCUSSION TITLE
Error occur when simulation executed via Rhino rasshopper

### DISCUSSION THREAD

**Discussion by yinlong-w** [urbanopt/urbanopt-cli#400] [Q&A]
![image](https://user-images.githubusercontent.com/100746242/210596212-7eb6ac37-4c02-4067-a8af-96bf9aeef7ab.png)
When i run the UrbanOpt via Rhino Grasshopper, the following error is met:
stderr: The process cannot access the file because it is being used by another process.
Therefore, how should i solve it as i cannot find any solution on the internet, thanks

**Comment by kflemin** [comment]
Hi @yinlong-w, For Rhino Grasshopper support, you may have better luck posting in the Ladybug Tools forum: https://discourse.ladybug.tools/
